cfme/automate/provisioning_dialogs.py

```diff
@@ -112,34 +67,77 @@ def type_nav(self):
         return self.type[0]
```

Comment:
i believe we should turn all nav type specification from a plain tuple to a namedtuple with the attributes nav and tree_nav